src/style.css

```
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  line-height: 300px;
+  flex-grow: 1;
+  color: white;
+  text-align: center;
+}
+
+.block-1 {
+  background-color: red;
+}
+
+.block-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block-6 {
+  background-color: black;
+}
+
+@media (min-width: 500px) and (max-width: 850px) {
```

```suggestion
@media (min-width: 600px) and (max-width: 899px) {
```
або частину and зовсім забрати, а min з 600